@xen-orchestra/lite/scripts/release.mjs

```diff
@@ -4,6 +4,7 @@ import argv from 'minimist'
 import { tmpdir } from 'os'
 import { fileURLToPath, URL } from 'url'
 import { $, cd, chalk, fs, path, question, within } from 'zx'
+import { readFileSync } from 'fs'
```

Comment:
You can use `fs.readFile` from the [`zx`](https://google.github.io/zx/api#fs) library.